Automattic/jetpack
# Forms: Multiple Choice and Single Choice fields redesign

## Proposed changes:
* Redesign the Multiple Choice and Single Choice fields to use Inner Blocks

### Other information:


## Jetpack product discussion
pcsBup-TA-p2

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

* Create a post and add a Form block
* Include a Multiple Choice and a Single Choice field
* Play with the blocks and check if they are working as expected
* Publish the page and test the published page
* Make sure to test the block migration by opening a page with an existent MS/SC field
* Check if the blocks were migrated properly
